src/extractor.rs

```diff
@@ -682,7 +808,7 @@ mod type_check_tests {
     #[test]
     fn recognizes_ref_source_config() {
         assert_all_type_check(vec![
-            "select * from {{ ref('my_table') }}",
+            "select * from {{ ref('my_table', v=2) }}",
```

Comment:
let's keep these tests the same and just copy paste the whole test for a "_with_ref_version" since it should still recognize both.